### DISCUSSION TITLE
How to eliminate the air resistance in airsim?

### DISCUSSION THREAD

**Discussion by Han-Sin** [microsoft/AirSim#3216] [Support Q&A] [answered]
I am doing a uav simulation in airsim, but now I don't want the air resistance... So how to make it?
I really need it...please.

**Comment by rajat2004** [comment, marked as answer]
You could try changing [this line](https://github.com/microsoft/AirSim/blob/master/AirLib/include/physics/Environment.hpp#L109) to set 0, haven't checked but probably shouldn't cause any division by 0 errors

**Comment by Han-Sin** [comment]
Thanks!
real work well
